src/pip/_internal/req/__init__.py

```diff
@@ -67,8 +71,12 @@ def install_given_reqs(
                 requirement.install(
                     install_options,
                     global_options,
-                    *args,
-                    **kwargs
+                    root=root,
+                    home=home,
+                    prefix=prefix,
+                    pycompile=pycompile,
```

Comment:
nit: it would be clearer if the argument order were the same in the call site and the definition.

Reply:
Done, using the same order as `InstallRequirement.install`.